talawa-mobile-docs/index.md

```diff
@@ -39,616 +35,304 @@
 <li>Visit the <a href="https://github.com/PalisadoesFoundation/talawa">Talawa GitHub</a> to see the code.</li>
 </ol>
 
-
 ## Libraries
 
 ##### [access_request_screen](views_after_auth_screens_join_org_after_auth_access_request_screen/views_after_auth_screens_join_org_after_auth_access_request_screen-library.md)
 
-
-
 ##### [access_request_view_model](view_model_access_request_view_model/view_model_access_request_view_model-library.md)
 
-
-
 ##### [add_members_bottom_sheet](widgets_add_members_bottom_sheet/widgets_add_members_bottom_sheet-library.md)
 
-
-
 ##### [add_post_page](views_after_auth_screens_add_post_page/views_after_auth_screens_add_post_page-library.md)
 
-
-
 ##### [add_post_view_model](view_model_after_auth_view_models_add_post_view_models_add_post_view_model/view_model_after_auth_view_models_add_post_view_models_add_post_view_model-library.md)
 
-
-
 ##### [app_localization](utils_app_localization/utils_app_localization-library.md)
 
-
-
 ##### [app_settings_page](views_after_auth_screens_app_settings_app_settings_page/views_after_auth_screens_app_settings_app_settings_page-library.md)
```

Comment:
why chhange this?

Reply:
The file contains references to the event chat and adheres to the Prettier formatter. Should I revert it?